examples/plugins/iowait/Makefile

```diff
@@ -0,0 +1,19 @@
+.PHONY: run clean
+
+EXE=iowait
+IMAGE=weavescope-iowait-plugin
+UPTODATE=.$(EXE).uptodate
+
+run: $(UPTODATE)
+	docker run --rm -it --privileged -v /var/run/scope/plugins:/var/run/scope/plugins --name $(IMAGE) $(IMAGE) -hostname=$(shell hostname)
```

Comment:
`/var/run/scope/plugins` won't exist if Scope is not running. I am not sure we want to do something about this. The http-requests plugin has this problem too.